terraform/terraform_backend.tf

```diff
@@ -1,6 +1,18 @@
 terraform {
-  backend "remote" {
-    organization = "nix-community"
-    workspaces { name = "nix-community" }
+  # will import resources into new project, won't transfer existing state
+  backend "s3" {
+    endpoint = "s3.nix-community.org"
```

Comment:
Don't know how much state churn we deal with, but I am also now quite happily using the encrypted state backend that can be committed to git. Works fine for small projects.

Reply:
Removing dns means there isn't much churn apart from occasional changes to hydra so committing state would probably be fine for this repo but I'm hoping we can also use it for https://github.com/ipdxco/github-as-code.

IIRC opentofu is still going to implement OCI for state storage so s3 may only be an interim solution anyway.